apps/docs/src/components/PropValues.js

```diff
@@ -1,13 +1,31 @@
 import React from 'react'
 import InlineCode from '@primer/gatsby-theme-doctocat/src/components/inline-code'
 
-export function PropTableValues({values, addLineBreaks}) {
+export function PropTableValues({
+  values,
+  addLineBreaks,
+  commaSeparated,
+  removeApostrophes,
+}) {
   const valuesToRender = values.map((value) => {
     if (typeof value === 'string') {
+      if (removeApostrophes) {
+        return <InlineCode key={value}>{value}</InlineCode>
+      }
```

Comment:
Duplicates line 17 below, perhaps consolidate the conditionals?

Reply:
Thanks, fixed 👍